content/blog/2022-02-25-react-v18-rc.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "React v18.0 Release Candidate: Upgrade Guide"
+author: [rickhanlonii]
+---
+
+Today we’re publishing the second release candidate for React 18. With this release, we’re encouraging users to install and test React 18 to [raise any issues](https://github.com/facebook/react/issues) for any problems you encounter in the upgrade before we publish the full stable release.
```

Comment:
nit: since this is about "raising issues" let's directly link the "new issue" link:
```suggestion
Today we’re publishing the second release candidate for React 18. With this release, we’re encouraging users to install and test React 18 to [raise any issues](https://github.com/facebook/react/issues/new/choose) for any problems you encounter in the upgrade before we publish the full stable release.
```

Ideally we could link the "React 18" issue but that would hardcode the templates, labels etc. So once you change the template or label, the link in this repo would be outdated.